backend/src/test/java/de/neuefische/simplesportsclub/backend/service/MessageServiceTest.java

```diff
@@ -129,4 +128,70 @@ void addNewMessageToGroup() {
         assertThat(actual, is(List.of(newMessage)));
 
     }
+    @Test
+    void removeMessageFromGroup(){
+
+        //GIVEN
+        Group newGroup = Group.builder()
+                .id("1")
+                .title("some-title")
+                .description("lets play some-sport")
+                .sport("some-sport")
+                .dateStart("2021-01-01 01:00:00.000")
+                .dateEnd("2021-01-01 02:00:00")
+                .participationFee(1)
+                .maxAmountOfParticipants(10)
+                .region(HAMBURG)
+                .location("some-address")
+                .messages(List.of(
+                        Message.builder()
+                                .username("some-principal-name")
+                                .messageText("some-message")
+                                .id("1")
+                                .build(),
+                        Message.builder()
+                                .username("some-principal-name")
+                                .messageText("some-message")
+                                .id("2")
+                                .build()
+                ))
+                .build();
+
```

Comment:
use @With annotation on your Group model and then
```suggestion
Group updatedGroup = newGroup.withMessages(List.of(messages))
```